### DISCUSSION TITLE
Build error related to minimum deployment target

### DISCUSSION THREAD

**Discussion by ccodenam** [codemagic-ci-cd/codemagic-docs#1878] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

I came to do a build today and I get an error regarding dependency versions of a flutter package I am using called `app_settings`. All of my previous builds were working so this is unexpected. I tried to make it work by adding a `pod repo update` script in my codemagic.yaml file, I still get the error though.

I am a paying customer but I don't see the in app chat widget on the website, so I had to post here :(

Here's the error
```
Adding spec repo `trunk` with CDN `https://cdn.cocoapods.org/`
  CDN: trunk Relative path downloaded: CocoaPods-version.yml, save ETag: "64b1752d-38"
  CDN: trunk Relative path: CocoaPods-version.yml exists! Returning local because checking is only performed in repo update
[!] CocoaPods could not find compatible versions for pod "app_settings":
  In Podfile:
    app_settings (from `.symlinks/plugins/app_settings/iOS`)

    Specs satisfying the `app_settings (from `.symlinks/plugins/app_settings/ios`)` dependency were found, but they required a higher minimum deployment target.
```

Some additional info. In my pod file the following lines were always commented out, and the build worked fine:

```
# platform :ios, '13.0'
```

After reading for solutions to my problem, I uncommented this line just to test if this made a different, it didn't, the build fails whether this line is commented or not.

### Expected results

Expected iOS to build.

### Actual results

Got the error mentioned above

### Build id (optional)

64b176bbc2c9dfb6b92047f8 <-- this was the first build to fail

64b17fe80f7f7cc9da9bf42a <-- this build included the `pod repo update` script, build still fails

**Comment by icarusdust** [comment]
Hi @brxxxx 

1. For quicker responses, please contact our support team via the chat widget in the UI enabled for your team that is on a paid plan 
2. Better let always `platform :ios, '13.0`' be commented in  in the Podfile, it is recommended in general.
3. Check in Xcode what deployment target you have set. You can check it by going to `ios/Runner.xcodeproj/project.pbxproj` and search  for `IPHONEOS_DEPLOYMENT_TARGET` see if it is 13.0
4. If it is 13.0 in all 3 lines in `project.pbxproj`, then the dependency might require higher version than 13.0. You can upgrade it and let me know if it helps.

**Comment by ccodenam** [comment]
Updating the `app_settings` package in the pub spec.yaml to version 5 seems to have fixed the problem @icarusdust
